spec/classes/puppet_agent_spec.rb

```diff
@@ -61,6 +61,7 @@
             .with_ensure('present')
             .with_provider(package_provider)
             .with_source(nil)
+	    .with_install_options(nil)
```

Comment:
```suggestion
            .with_source(nil)
            .with_install_options(nil)
```

It looks correct in my editor but GH shows the alignment as off. Converting tabs to spaces here should fix that :)